.github/workflows/pylint.yml

```
@@ -9,7 +9,7 @@ jobs:
       matrix:
         python-version: ["3.10"]
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v3
```

actionlint (1.7.7)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)